documentation/hpcc-tf-for-developers.md

```diff
@@ -0,0 +1,373 @@
+| `authn_htpasswd_filename` | string | If you would like to use htpasswd to authenticate users to the cluster, enter the filename of the htpasswd file.  This file should be uploaded to the Azure 'dllsshare' file share in order for the HPCC processes to find it. A corollary is that persistent storage is enabled. An empty string indicates that htpasswd is not to be used for authentication. Example entry: "htpasswd.txt" |
+| `enable_code_security` | boolean | Enable code security? If true, only signed ECL code will be allowed to create embedded language functions, use PIPE(), etc. Example entry: false |
+| `enable_premium_storage` | boolean | If true, premium ($$$) storage will be used for the following storage shares: Dali. OPTIONAL, defaults to false. |
+| `enable_thor` | boolean | If you want a thor cluster then 'enable_thor' must be set to true Otherwise it is set to false |
+| `external_storage_desired` | boolean | If you want external storage instead of ephemeral storage then set this variable to true otherwise set it to false. |
+| `extra_tags` | map of string | Map of name => value tags that can will be associated with the cluster. Format is '{"name"="value" [, "name"="value"]*}'. The 'name' portion must be unique. To add no tags, use '{}'. |
+| `hpcc_user_ip_cidr_list` | list of string | List of explicit CIDR addresses that can access this HPCC Systems cluster. To allow public access, specify "0.0.0.0/0". To add no CIDR addresses, use '[]'. |
+| `hpcc_version` | string | The version of HPCC Systems to install. Only versions in nn.nn.nn format are supported. |
+| `my_azure_id` | string | Your azure account object id. Find this on azure portal, by going to 'users' then search for your name and click on it. The account object id is called 'Object ID'. There is a link next to it that lets you copy it. |
+| `storage_data_gb` | number | The amount of storage reserved for data in gigabytes. Must be 1 or more. If a storage account is defined (see below) then this value is ignored. |
+| `storage_lz_gb` | number | The amount of storage reserved for the landing zone in gigabytes. Must be 1 or more. If a storage account is defined (see below) then this value is ignored. |
+| `thor_max_jobs` | number | The maximum number of simultaneous Thor jobs allowed. Must be 1 or more. |
+| `thor_num_workers` | number | The number of Thor workers to allocate. Must be 1 or more. |
+
+
+The following table gives the name of each of the 5 `null_resource` in `main.tf` and gives a short description of what each does.
+
+|null_resource name|description|
+|:-----------------|:----------|
+| `deploy_vnet` | deploys aks' virtual network |
+| `deploy_aks` | deploys aks |
+| `deploy_storage` | deploys persistent storage |
+| `external_storage` | waits for deployment of presistent storage |
+| `deploy_hpcc` | deploys hpcc |
+
+The subfolders, except for `scripts`, create components needed by the full system.
+
+## scripts subdirectory
```

Comment:
Most of the 'here are the contents of the subdirectory' sections appear to be out of date.  Please update them.

Reply:
The statemeint, 'here are the contents of the subdirectory', no longer exists.

Reply:
Fixed